features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml

```diff
@@ -1534,4 +1534,16 @@
                    description="Delete user defined local authenticator"/>
         </Scopes>
     </APIResource>
+    <APIResource name="User defined local authenticator API" identifier="/o/api/server/v1/authenticators/custom"
+                 requiresAuthorization="true"
+                 description="API representation of the User Defined Local Authenticator Management API" type="ORGANIZATION">
+        <Scopes>
+            <Scope displayName="Create Authenticator" name="internal_custom_authenticator_create"
```

Comment:
```suggestion
            <Scope displayName="Create Authenticator" name="internal_org_custom_authenticator_create"
```

Reply:
let's update other B2B API scopes with `internal_org_` prefix

Reply:
Let's add `internal_org_` prefix for organization API scopes. Check other places as well